views.md

```diff
@@ -1,7 +1,88 @@
 # Views
 
+A view is a window into your skill. It can be rendered as an `iframe` in a browser or as `UIWebView` on iOS (Android coming 2020'ish).
+
+On most pages, Views are accessible through tabs:
+
+![Skill Views](_images/skill-views.png?raw=true "Skill Views")
+
+Pages are built using the [Heartwood](#heartwood) design system (which is totally ready for you to play with at [https://storybook.spruce.ai].)
+
 ## Skill View Pages
 
+When you use `sprucebot skill create` as your starting point, you'll have a ton of placeholder view pages to get you started. Ultimately, rendering a skill view is a 3 steps to setting up a Skill View.
+
+### Step 1: Event contracts
+
+Uncomment the following block:
+
```

Comment:
Say where the block lives (config/default.js), and maybe _why_ eg: "We need to configure the skill to listen for skill views."